cclib/bridge/cclib2horton.py

```diff
@@ -82,7 +82,40 @@ def makehorton(ccdat):
             attributes["npa_charges"] = ccdat.atomcharges["natural"]
 
     elif hortonver == 3:
-        pass  # Populate with bridge for horton 3 in later PR
+        # In horton 3 IOData, inputs are type-checked -- thus the bridge also verifies the types.
+        # if coordinates are known; numpy.ndarray (size natom-by-3)
+        if hasattr(ccdat, "atomcoords") and isinstance(ccdat.atomcoords, numpy.ndarray):
+            attributes["atcoords"] = ccdat.atomcoords[-1]
+        # if atomic numbers are known; numpy.ndarray (size natom)
+        if hasattr(ccdat, "atomnos") and isinstance(ccdat.atomnos, numpy.ndarray):
+            attributes["atnums"] = ccdat.atomnos
+        # if orbital coefficients known; iodata.orbitals.MolecularOrbitals
+        if hasattr(ccdat, "mocoeffs") and isinstance(ccdat.mocoeffs, numpy.ndarray):
+            attributes["mo"] = MolecularOrbitals(
+                kind="restricted",
+                norba=len(ccdat.mocoeffs[0]),
+                coeffs=ccdat.mocoeffs[0].T,
+            )
+            # and if unrestricted:
+            if len(mocoeffs) == 2:
+                attributes["mo"].kind = "unrestricted"
+                attributes["mo"].norbb = len(ccdat.mocoeffs[1])
+                attributes["mo"].coeffs.append(ccdat.mocoeffs[1].T)
+        # if multiplicity known; float / should not be set when mocoeffs present
```

Comment:
Can you explain why?

Reply:
I think it's meant so that the multiplicity is derived from MO and rather not be overridden by the `spinpol` attribute. This is mentioned in `IOData` [code](https://github.com/theochem/iodata/blob/b36513d162f99b57264005583701c6987037839c/iodata/iodata.py#L174). I'll add this as comment for reference. Thanks.